src/modules/tests/__mocks__/tests.mock.ts

```diff
@@ -24,20 +68,20 @@ export const candidate: Candidate = {
   candidateName: {
     firstName: 'Practice',
     lastName: 'Mode',
-    title: 'Miss',
+    title: 'End to End',
```

Comment:
We don't usually show candidate name with the title so if this is something you want to display it needs to be first or last name